### ISSUE TITLE
Add-ons: Warn when starting NVDA if any imcompatible add-ons can't be loaded

### ISSUE THREAD

**Issue by elliott94** [nvaccess/nvda#11643]
### Is your feature request related to a problem? Please describe.

Consider the following:

1. A user is using a pre-2019.3 build of NVDA, and receives a new computer - so installs NVDA 2020.2 onto it

2. The user would like to retain their existing config, so copies the "nvda" directory from their old machine to %appdata% on the new one, at which point they restart NVDA. It's worth noting that their old config contains several add-ons and synth drivers that are not built for Python 3, and as a result won't work

Upon restarting NVDA, due to the fact that their preferred synth with their configuration was based on a synth driver that doesn't work with Python 3, NVDA refuses to load it - and instead switches to an alternative synth that does work. Whilst errors are written to the log, due to the fact that this is a production version of NVDA, no error sound is output.

In these cases, when simply copying over an existing configuration after installation of NVDA, could we consider presenting the Incompatible Add-ons Manager (if any incompatible add-ons are detected) to minimise confusion and to give users an explanation as to why their previous add-ons will no longer be loaded in newer NVDA versions?

**Comment by bhavyashah** [comment]
An notable feature of this ticket is that its importance will considerably decrease overtime. As we move into 2021 and beyond, users upgrade to recent NVDA releases, and add-on authors catch up on compatibility, incompatible add-ons will become less and less relevant. Perhaps there still is a significant user base of NVDA 2019.3 and earlier who may experience the described scenario on updating making this ticket relevant for now. I also think this issue would affect users who revisit an old computer with an old copy of NVDA, though I am unclear on how sizeable that group would be.
